kit/Kit.cpp

```diff
@@ -1939,13 +1940,26 @@ std::shared_ptr<lok::Document> Document::load(const std::shared_ptr<ChildSession
             | LOK_FEATURE_VIEWID_IN_VISCURSOR_INVALIDATION_CALLBACK;
         _loKit->setOptionalFeatures(flags);
 
+        std::string loadUri = uri;
+
+        if (!docTemplate.empty())
```

Comment:
Could you please add a comment here on how this works? Here is how I understand it (but probably I'm wrong):

1) 'uri' is the new document, say doc.pptx and 'docTemplate' is the template, say template.potx
2) now you tweak newPath to be be doc.potx instead of doc.pptx.

Shouldn't this be the other way around? Thanks.

Reply:
This is done now, thanks.